src/main/fc/fc_tasks.c

```diff
@@ -0,0 +1,485 @@
+/*
```

Comment:
Not strictly needed. However if OSD is ported from Cleanflight then we will have OSD tasks, so this anticipates that.